package.json

```diff
@@ -64,9 +64,7 @@
     "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "^12.11.1",
-    "@typescript-eslint/eslint-plugin": "4.16.1",
-    "@typescript-eslint/parser": "4.16.1",
-    "eslint": "^7.6.0",
+    "codelyzer": "^6.0.0",
```

Comment:
Could we undo this part? This repo is very specifically migrated to eslint...

Reply:
Done. Thanks!